components/brave_wallet_ui/components/desktop/portfolio-account-item/index.tsx

```diff
@@ -1,51 +1,55 @@
 import * as React from 'react'
 import { create } from 'ethereum-blockies'
+import { useHistory } from 'react-router'
+
+// Types
+import { BraveWallet, DefaultCurrencies, WalletAccountType, WalletRoutes } from '../../../constants/types'
 
 // Hooks
 import { useExplorer, usePricing } from '../../../common/hooks'
 
 // Utils
 import { reduceAddress } from '../../../utils/reduce-address'
 import Amount from '../../../utils/amount'
-
 import { getLocale } from '../../../../common/locale'
-import { BraveWallet, DefaultCurrencies } from '../../../constants/types'
+
+// Components
 import { TransactionPopup, WithHideBalancePlaceholder } from '../'
 import { CopyTooltip } from '../../shared/copy-tooltip/copy-tooltip'
+import { TransactionPopupItem } from '../transaction-popup'
 
 // Styled Components
 import {
   StyledWrapper,
   AssetBalanceText,
-  AccountName,
-  AccountAddress,
+  AccountNameButton,
+  AccountAddressButton,
   AccountAndAddress,
   BalanceColumn,
   FiatBalanceText,
   NameAndIcon,
   AccountCircle,
   MoreButton,
   MoreIcon,
-  RightSide
+  RightSide,
+  CopyIcon,
+  AddressAndButtonRow
 } from './style'
-import { TransactionPopupItem } from '../transaction-popup'
 
-export interface Props {
+interface Props {
   spotPrices: BraveWallet.AssetPrice[]
+  account: WalletAccountType
```

Comment:
It's generally a bad practice to pass a large object as a prop when only a couple of the objects properties are being consumed. Any time an account's balance registry is changed will also cause this to re-render.